### PR TITLE
Make deletions show like timeouts/bans.

### LINKED ISSUE #1155: Deleted messages wont show like timeout
Example: if a mod deletes a message it will show: "The message from -username- is now deleted" only for the mod that deleted the message. But it wont show for the other mods, the message will just get deleted with no message like a timeout.

### PR DESCRIPTION
**Pull request by Mm2PL**
Pull request checklist:

- [x] `CHANGELOG.md` was updated, if applicable

# Description

Closes #1155.
Adds a notice for when a message gets deleted.

### CODE REVIEW

**Comment by Mm2PL** [comment]
###### lidl review request pajaDink
*Mm2PL requested review from @zneix and @pajlada*

File: src/providers/twitch/TwitchMessageBuilder.cpp [hunk 1]
```diff
@@ -1356,5 +1356,36 @@ void TwitchMessageBuilder::hostingSystemMessage(const QString &channelName,
...
+    // TODO(mm2pl): If or when jumping to a single message gets implemented a link,
+    // add a link to the originalMessage
+    builder->emplace<TextElement>("A message from", MessageElementFlag::Text,
+                                  MessageColor::System);
+    builder
+        ->emplace<TextElement>(originalMessage->displayName,
+                               MessageElementFlag::Username,
+                               MessageColor::System, FontStyle::ChatMediumBold)
+        ->setLink({Link::UserInfo, originalMessage->loginName});
+    builder->emplace<TextElement>("was deleted: ", MessageElementFlag::Text,
```
Comment (pajlada) [inline src/providers/twitch/TwitchMessageBuilder.cpp:1375]: The suffix space here is extraneous and causes it to show up as a double space in Chatterino

image

```suggestion
    builder->emplace<TextElement>("was deleted:", MessageElementFlag::Text,
```

**Comment by pajlada** [comment]
Behaviour in web chat:
image

Behaviour with this PR:
image

The extraneous space will be fixed, but other than that it looks good :ok_hand:

**Comment by Mm2PL** [comment]
Closes #2867

**Review by pajlada — APPROVED** [review]
LGTM

**Review by zneix — APPROVED** [review]

**Pull request merged**